test/e2e/e2e-jib.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+source ./test/helper/helper.sh
+
+# E2E tests parameters for the test pipeline
+
+# Testing the jib-maven task,
+@test "[e2e] jib-maven task" {
+    
+    run tkn pipeline start task-jib-maven \
+        --param="URL=${E2E_MAVEN_PARAMS_URL}" \
+        --param="REVISION=${E2E_MAVEN_PARAMS_REVISION}" \
+        --param="VERBOSE=true" \
+        --param="IMAGE=${E2E_JIB_MAVEN_IMAGE}" \
+        --workspace="name=source,claimName=task-maven,subPath=source" \
+        --workspace="name=dockerconfig,secret=docker-config" \
```

Comment:
where did we create `docker-config` secret :thinking:

Reply:
oh ya that was locally when i was testing, changing that.